core/clingutils/res/TClingUtils.h

```diff
@@ -197,28 +197,38 @@ class AnnotatedRecordDecl {
    std::string fRequestedName;
    std::string fNormalizedName;
    std::string fDemangledTypeInfo;
+   // clang-format off
    bool fRequestStreamerInfo;
    bool fRequestNoStreamer;
    bool fRequestNoInputOperator;
    bool fRequestOnlyTClass;
    int  fRequestedVersionNumber;
+   int  fRequestedRNTupleSplitMode;
```

Comment:
```suggestion
   bool  fRequestedRNTupleSplitMode;
```
isn't it?

At to same some space I would move it next to the other bool make then bitfields:
```
   bool  fRequestedRNTupleSplitMode : 1;
```

Reply:
humm ... nevermind it actually has 3 values.